wrappers/rust/icicle-core/src/ntt/mod.rs

```diff
@@ -77,6 +77,8 @@ pub struct NTTConfig<'a, S> {
     pub coset_gen: S,
     /// The number of NTTs to compute. Default value: 1.
     pub batch_size: i32,
+    /// True if the NTTs are stored as the columns of a matrix and not as the rows.
```

Comment:
I would rephrase and say that when true, NTT is computed over columns.

Reply:
done